tox.ini

```
@@ -1,26 +1,25 @@
 [tox]
-envlist = python{3.8}-django{22,30,31,32},quality
+envlist = python{3.8}-django{32,40},quality
 
 [testenv]
 passenv = CI TRAVIS TRAVIS_*
 commands = python -Wd -m pytest {posargs}
 deps = 
-	-r{toxinidir}/requirements/test.txt
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
+    -r{toxinidir}/requirements/test.txt
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<4.1
+    
 
 [testenv:quality]
 whitelist_externals = 
-	make
-	rm
-	touch
+    make
+    rm
+    touch
 deps = 
-	-r{toxinidir}/requirements/quality.txt
+    -r{toxinidir}/requirements/quality.txt
 commands = 
-	pylint djpyfs
-	isort --check-only --recursive django-pyfs
-	python setup.py sdist bdist_wheel
-	twine check dist/*
+    pylint djpyfs
+    isort --check-only --recursive django-pyfs
```

`recursive` flag can be removed now.
```suggestion
    isort --check-only django-pyfs
```